README.md

```diff
@@ -35,10 +35,10 @@ Since November 2018, Max Prokhorov (**@mcspr**) is also actively working as a co
 
 > Please use the [gitter ESPurna channel](https://gitter.im/tinkerman-cat/espurna) for support and questions, you have better chances to get fast answers from me or other ESPurna users. Open an issue here only if you feel there is a bug or you want to request an enhancement. Thank you.
 
-> Process of building ESPurna from source is documented in the Wiki:  
-[Using PlatformIO IDE for VSCode](https://github.com/xoseperez/espurna/wiki/Build-and-update-from-Visual-Studio-Code-using-PlatformIO)  
-[Using PlatformIO CLI](https://github.com/xoseperez/espurna/wiki/PlatformIO)  
-[Using Arduino IDE](https://github.com/xoseperez/espurna/wiki/ArduinoIDE)  
+> Process of building ESPurna from source is documented in the Wiki:
+[Using PlatformIO IDE for VSCode](https://github.com/xoseperez/espurna/wiki/Build-and-update-from-Visual-Studio-Code-using-PlatformIO)
+[Using PlatformIO CLI](https://github.com/xoseperez/espurna/wiki/PlatformIO)
+[Using Arduino IDE](https://github.com/xoseperez/espurna/wiki/ArduinoIDE)
```

Comment:
pls take a look at the resulting render
`<double-space>` is intentional to create separate lines